API/fleece/Fleece.h

```diff
@@ -368,6 +368,14 @@ extern "C" {
     void FLSlot_SetData(FLSlot FLNONNULL, FLSlice) FLAPI;    ///< Stores a data blob into a slot.
     void FLSlot_SetValue(FLSlot FLNONNULL, FLValue) FLAPI;   ///< Stores an FLValue into a slot.
 
+    static inline void FLSlot_SetArray(FLSlot NONNULL slot, FLArray array) {
```

Comment:
I wonder if array should be non-null.  Would a null array be a shortcut for `SetNull` ?